app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -427,6 +428,7 @@ public boolean onBackPressed() {
             SessionManager.getInstance().removeCurrentSession();
         }
 
+        TelemetryWrapper.controllerBackEvent(ControllerBackLocation.BROWSER);
```

Comment:
How does this interact with the "esc" case in Youtube? How does it count "closing the sidebar"? Do we actually want to count the *combined* number of times people go back in web history + navigate through the app? I think that this should either be two events (one for pop, one for browser back) or just put it in 'canGoBack'.

My concern is that this will not be a useful probe because it aggregates data from too many actions. Better to be specific.

Reply:
I changed this probe – please see the new commit and the commit summary for an explanation.

Barbara said I should ignore youtube ESC for now.